### PR TITLE
Prevents hallucinations revealing people's identity

### PR DESCRIPTION
**Pull request by XDTM**
They currently use the basic compose_message proc, which means that it appears as `Mike Traitor (as Clever Disguise) says, "stuff"`

Bug found by @kevinz000

### CODE REVIEW

**Comment by RemieRichards** [comment]
you mean "Bug abused by @kevinz000 until it was abused against him meaning it must now perish"

File: code/game/say.dm [hunk 1]
```diff
@@ -44,7 +44,7 @@ GLOBAL_LIST_INIT(freqtospan, list(
 /atom/movable/proc/get_spans()
 	return list()
 
-/atom/movable/proc/compose_message(atom/movable/speaker, datum/language/message_language, raw_message, radio_freq, list/spans, message_mode)
+/atom/movable/proc/compose_message(atom/movable/speaker, datum/language/message_language, raw_message, radio_freq, list/spans, message_mode, face_name = FALSE)
```
Comment (Cyberboss) [inline code/game/say.dm]: Make sure you update all overrides of compose_message

Comment (XDTM) [reply code/game/say.dm]: There are none!

**Comment by kevinz000** [comment]
@RemieRichards owie

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**